bridge/actions/Stats.cpp

```diff
@@ -0,0 +1,18 @@
+#include "ApiActions.h"
```

Comment:
Can we rename this file. We have too many Stats.cpp. The others are named Get....